.github/workflows/ci.yaml

```diff
@@ -4,16 +4,23 @@ on: [push]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
Remind me to talk to you about this ci process.  I want to ask a couple questions.